.github/workflows/reusable_test.yml

```diff
@@ -95,10 +104,10 @@ jobs:
         uses: ./.github/actions/common_setup
         with:
           job_type: test
-      - name: Download json reports
-        uses: actions/download-artifact@v3
-        with:
-          path: ${{ env.REPORTS_PATH }}
+      # - name: Download json reports
+      #   uses: actions/download-artifact@v3
+      #   with:
+      #     path: ${{ env.REPORTS_PATH }}
```

Comment:
This thing is so far too wide. We'll get rid of it at the moment we're 146% sure that GH artifacts aren't used.

It doesn't fail on error